salt/orch/migration.sls

```diff
@@ -0,0 +1,85 @@
+{#- Get a list of nodes seem to be down or unresponsive #}
```

Comment:
`migration.sls` gives the impression this orchestration is  general cluster migration orochestration, while is it focused on migrating just the `transactional-update` stuff. I would rename this file, so it is more clear this only affects the `transactional-update`...

Reply:
hmm not sure what you mean with "migrating just the transactional-update stuff". this orch is the actual migration (installation) from version to next version. and the tool to do it is `transactional-update`

Reply:
then I don't understand what is the difference between the `migration` and the `update` orchestrations...

Reply:
I think I understand @inercia point, because the `orchestration` doesn't actually *do* the migration, but just prepares the node to be ready to take the correct steps to do the actual upgrade:

1. Admin - only needs a reboot

2. Cluster nodes -  need to use the update orchestration

Maybe we should rename it to something like `migration_bootstrap` or `migration_preparation` instead?

Reply:
Yes, I wouldn't use a generic name for something that is not really generic (it seems more like a corner case or a previous step necessary for doing other things)...

Reply:
renamed to `prepare-product-migration`